compiler/src/dmd/parse.d

```diff
@@ -939,7 +939,7 @@ class Parser(AST, Lexer = dmd.lexer.Lexer) : Lexer
                         if (e)
                             error("redundant alignment attribute `align(%s)`", e.toChars());
                         else
-                            error("redundant alignment attribute `align`");
+                            error("redundant alignment attribute `align(default)`");
```

Comment:
you will need to update `fail_compilation/parseStc2.d` for this

Reply:
```diff
diff --git a/compiler/test/fail_compilation/parseStc2.d b/compiler/test/fail_compilation/parseStc2.d
index 936769a59d..dbc98f91ca 100644
--- a/compiler/test/fail_compilation/parseStc2.d
+++ b/compiler/test/fail_compilation/parseStc2.d
@@ -53,10 +53,10 @@ public private void f10() {}
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/parseStc2.d(63): Error: redundant alignment attribute `align`
+fail_compilation/parseStc2.d(63): Error: redundant alignment attribute `align(default)`
 fail_compilation/parseStc2.d(64): Error: redundant alignment attribute `align(1)`
 fail_compilation/parseStc2.d(65): Error: redundant alignment attribute `align(1)`
-fail_compilation/parseStc2.d(66): Error: redundant alignment attribute `align`
+fail_compilation/parseStc2.d(66): Error: redundant alignment attribute `align(default)`
 fail_compilation/parseStc2.d(67): Error: redundant alignment attribute `align(2)`
 ---
 */
```